picard/coverart/image.py

```diff
@@ -324,6 +324,16 @@ def set_tags_data(self, data):
     def set_external_file_data(self, data):
         self.external_file_coverart = CoverArtImage(data=data, url=self.url)
 
+    @property
+    def has_finished_processing(self):
+        if not self.datahash:
+            return False
+        config = get_config()
+        if (config.setting['save_images_to_files'] and not
+                (self.external_file_coverart and self.external_file_coverart.datahash)):
```

Comment:
Perhaps it would be better to use call `get_config()` only if strictly needed and do:

```python
if not (self.external_file_coverart and self.external_file_coverart.datahash):
    config = get_config()
    if config.setting['save_images_to_files']:
        return False
return True
```